docs/source/conf.py

```diff
@@ -21,7 +21,7 @@
 sys.path.insert(0, os.path.abspath("../.."))
 
 project = "movement"
-copyright = "2022, UCL"
+copyright = "2023, University College London"
```

Comment:
Do you want to do something like:

```python
from datetime import date

copyright = f"2022–{date.today().year}, University College London"
```

Reply:
I'm actually not sure how copyright years work, is it supposed to be the date of creation, or always the current date? Also how does it work with versioned products, like software? Tagging our resident IP specialist @adamltyson 😉

Reply:
After some digging, specifying a year range should be appropriate, so you were right @lochhh

Reply:
I usually just add the year the software was first released. IIRC copyright is automatic in the UK, so it doesn't really make much difference anyway. I would just set it to be 2023 and leave it for simplicity.